client/post-editor/media-modal/detail/test/index.jsx

```diff
@@ -11,6 +15,15 @@ jest.mock( 'calypso/post-editor/media-modal/detail/detail-file-info', () =>
 	require( 'calypso/components/empty-component' )
```

Comment:
I think this mock can be removed - it doesn't seem necessary, as long as we have the above mock.